aws_lambda/aws_lambda.py

```diff
@@ -348,29 +348,35 @@ def create_function(cfg, path_to_zip_file):
     client = get_client('lambda', aws_access_key_id, aws_secret_access_key,
                         cfg.get('region'))
 
+    #Do we prefer development variable over config?
```

Comment:
Great question, I poked around some other projects and it seems like the defacto standard is enviroment variables should take precedence over configuration vars.